app/controllers/concerns/caching_headers.rb

```diff
@@ -13,29 +13,21 @@ module CachingHeaders
   #  {cache_control: 'public, no-cache, maxage=xyz', surrogate_control: 'max-age: 100'}
   def set_cache_control_headers(
     max_age = 1.day.to_i,
-    surrogate_control: nil, stale_while_revalidate: nil, stale_if_error: 26_400
+    surrogate_control: nil,
+    stale_while_revalidate: nil,
+    stale_if_error: 26_400
   )
-
-    # TODO: [@forem/delightful]: Once we've tested that removing `no-cache` from the Cache-Control headers
-    # does not adversely affect fastly, we should remove this specific check entirely.
-    article_requested = request.env["REQUEST_PATH"].present? &&
-      request.env["REQUEST_PATH"].include?("the-3-biggest-misconceptions-about-diversity-3hkm")
-    cache_control = if article_requested
-                      "public, max-age=86400"
-                    else
-                      "public, no-cache"
-                    end
-
     request.session_options[:skip] = true # no cookies
-    response.headers["Cache-Control"] = cache_control
 
+    response.headers["Cache-Control"] = "public, no-cache" # Used only by Fastly.
+    response.headers["X-Accel-Expires"] = max_age.to_s # Used only by Nginx.
```

Comment:
Nice solution. Question, bc we took the `max-age=86400` out of the cache_control headers for articles will that affect how we are caching them now? They will still expire, correct? Sorry if that is a dumb question 😅

Reply:
@mstruve Hm, the only article that we experimented on was [that one article](https://github.com/forem/forem/pull/9655/files#diff-09e7aa268f812ebaaa34e1ff19996cebL21-L23) of @atsmith813's right? Otherwise, we didn't touch any other article's headers 🤔 

`max-age` will still get sent through to Fastly [via the Surrogate-Control header](https://github.com/forem/forem/blob/36bb896caf94e48e644b5be55904f66ef9f16f4e/app/controllers/concerns/caching_headers.rb#L39), so I don't think any endpoints should be affected with regards to caching on Fastly

(Let me know if I am understanding your question correctly, I have a feeling I might not be? 🙃)

Reply:
OH right, nvm ignore me. It all just came together in my head

Reply:
🙌